packages/module/src/Ansible/Ansible.tsx

```diff
@@ -45,18 +45,18 @@ const useStyles = createUseStyles({
   },
   ansibleUnsupported: {
     '& .st0, .st1, .st2': {
-      fill: 'var(--pf-t--color--gray--50)',
+      fill: 'var(--pf-t-global--icon--color--disabled)',
       cursor: 'not-allowed',
     }
   }
 })
 
-const Ansible: React.FunctionComponent<AnsibleProps> = ({ isUnsupported, isRHAAP, className, ouiaId = "Ansible-icon", ...props }: AnsibleProps) => {
+const Ansible: React.FunctionComponent<AnsibleProps> = ({ isSupported, isRHAAP, className, ouiaId = "Ansible-icon", ...props }: AnsibleProps) => {
```

Comment:
```suggestion
const Ansible: React.FunctionComponent<AnsibleProps> = ({ isSupported = true, isRHAAP, className, ouiaId = "Ansible-icon", ...props }: AnsibleProps) => {
```

We should make the isSupported `true` by default